File: mvx/mvc/gin-mvc/pkg/response/response.go
package response

import (
  "github.com/gin-gonic/gin"
)

// StandardResponse 统一的 JSON 响应格式
func StandardResponse(c *gin.Context, status int, data interface{}, message string) {
  c.JSON(status, gin.H{
    "code":    status,
    "data":    data,
    "message": message,
  })
}

// SuccessResponse 统一的成功响应
func SuccessResponse(c *gin.Context, status int, data interface{}) {
  StandardResponse(c, status, data, "成功")
}

// ErrorResponse 统一的错误响应
func ErrorResponse(c *gin.Context, status int, message string) {
  StandardResponse(c, status, nil, message)
}

File: practice-projects/flask-mvc/utils/response.py
# utils/response.py
from flask import jsonify

def api_response(data=None, message='success', code=0, status=200):
    """统一API响应格式"""
    return jsonify({
        'code': code,
        'message': message,
        'data': data,
        'success': code == 0
    }), status

def success_response(data=None, message='操作成功', code=0):
    """成功响应"""
    return api_response(data=data, message=message, code=code, status=200)

def error_response(message='操作失败', code=1, status=400):
    """错误响应"""
    return api_response(data=None, message=message, code=code, status=status)